packages/tx/src/kzg/trusted_setup.txt

```diff
@@ -0,0 +1,4163 @@
+4096
+65
+97f1d3a73197d7942695638c4fa9ac0fc3688c4f9774b905a14e3a3f171bac586c55e83ff97a1aeffb3af00adb22c6bb
+92a72fc2d2d7888d95bd8b479fd66555908ef365b31d181c6aae60b1cbbcc21110f057dd13f9b694b2b260cda1bcb516
+934c3d60d8a31e9cd617427b842dbd49d7dcc5c8949d932773f0a0d752786f229d7017c592b3303dc5f631d2c01d5c08
+a895d8f7f4f160ab3837d80b5f6aa743b28d22ed22116269f3857130709df107867ba7e0db0d5e171492fab0db2a635f
+82d3d5a542a525646953cf68517013508bf9aec3cd7faf0a311fb0970f07697c8efc6943fb7fb8e9ff208a30ce4e8027
+95e3d50c2686c92568f46c8aa3a0e7cb3d456cc532c4ea0d07d47dbbed678fca2976b0b28009a360a2fa3dcd2716a7dd
+b7b60fc315983545150f0b6545509ffe3f38b5562574e640dd9c61d1a5db441855df9b0cab10096fbca09024fe076602
+86836a32ca078c51b9e9012fd37cf6647bc4bc2b8331b8c51f0958aed321de1fe44c2ff374d34eba13fcac18f3061a63
+b50bd23db7116d7cdf19bb3bbf38d946f345c2b80e31bd3ac566efcb302561d3727df398ba913d7f3436b82003337c0c
+83298b2de8fc1e36765903cb94d1b3e45ea5a09f5e0468dd4978ccd9577f81e2985f98f7294a3233d822f54d8d3f35e7
+89c74ab72d45bce23aa7b71d063f0e24e03353909570ce9459e8da7f872880c0267774699a10e88ca8d169b7f906da30
```

Comment:
Ah, do we need to publish this with the final package?

Guess this is the final argument to do a separate `@ethereumjs/shardblob4844tx`

Reply:
Not sure yet.  Depends on whether the final kzg library we end up using (whether `c-kzg` or the rust equivalent) packages that trusted setup with its code or not.

Reply:
Ok, but for (extended) bundle size that would mean the same.

Reply:
I would assume so. We could probably compress it down but this is something like it will be required for us to be able to verify the proof associated with each blob.

Reply:
Yeah, just still thinking from the extract-an-own-4844-tx-or-not perspective...